src/java.management/share/classes/javax/management/remote/JMXConnector.java

```diff
@@ -175,9 +155,14 @@ public MBeanServerConnection getMBeanServerConnection()
      * deprecated and subject to removal. There is no replacement.
      */
     @Deprecated(since="21", forRemoval=true)
-    public MBeanServerConnection getMBeanServerConnection(
-                                               Subject delegationSubject)
-            throws IOException;
+    public default MBeanServerConnection getMBeanServerConnection(Subject delegationSubject)
```

Comment:
Specifying the UOE in the param description is problematic. I think you want to specify delegationSubject as not used, must be null, then declare UOE as one of the exceptions thrown by the method.

Reply:
Thanks,  updated.